voir/instruments/gpu/__init__.py

```diff
@@ -130,8 +127,7 @@ def _visible_devices(smi):
     return ours
 
 
-@instrument_definition
-def gpu_monitor(ov, poll_interval=10, arch=None):
+def gpu_monitor():
```

Comment:
The docstring for this function should be moved to the other gpu_monitor function that has the original signature.

Reply:
Done!